src/lib/shared/updateAction.ts

```diff
@@ -0,0 +1,7 @@
+// helper to update state from actions
+export default function updateAction(state: any, payload: any) {
```

Comment:
Not sure if this helper is necessary, its quite a simple function and adds complexity when reading code.

Reply:
Oh i see, it's used with `little-state-machine`. In that case, can it be named something more descriptive, such as `updateStateFromAction`?